### PR TITLE
adding slack channel information

### PR DESCRIPTION
**Pull request by shrunyan**

### CODE REVIEW

**Comment by hbrysiewicz** [comment]
Is this tbm? or still wip?

File: chat-room.md [hunk 1]
```diff
@@ -4,20 +4,30 @@ title: Chat Room
...
 {% include JB/setup %}
-Meetup events aren't the only time for chatting. There's also a #sandiegojs IRC channel on Freenode. Sign on to ask questions, give answers, or just hang out.
 
-This page contains instructions for signing onto the IRC channel.
+Keep in touch outside of the monthly meetups. Join one of the chat rooms to ask questions, give answers, or just hang out.
 
-## Web Client
+## Slack
+
+Signup for the [SDJS Slack Channel](https://sdjs.slack.com)
```
Comment (hbrysiewicz) [inline chat-room.md]: You can request an invite at http://sandiegojs.herokuapp.com

**Comment by shrunyan** [comment]
@hbrysiewicz updated PR to include link to the channel request. What does tbm mean?

**Comment by hbrysiewicz** [comment]
to be merged

**Comment by shrunyan** [comment]
I see. Yes it is ready TBM.

**Comment by treyhunner** [comment]
So we've killed off the IRC channel?  Some of us are still using it...

Should I make an announcement in it and change the topic?

**Comment by treyhunner** [comment]
When I click on the `#` link I can't sign in.  I assume I have to sign up first, right?

**Pull request merged**